modules/test/playwright/package.json

```diff
@@ -10,7 +12,15 @@
 	"license": "ISC",
 	"main": "index.js",
 	"name": "@liferay/playwright",
+	"private": true,
+	"resolutions": {
+		"prettier": "2.1.2"
+	},
 	"scripts": {
+		"checkFormat": "liferay-npm-scripts prettier \"**/*.ts\" --list-different --experimental-standalone",
+		"format": "liferay-npm-scripts prettier \"**/*.ts\" --write --experimental-standalone",
```

Comment:
Can we remove the experimental flag if we add support for checking a package-lock instead of yarn.lock in npm-scripts?

Thinking that it works fine in `./workspaces`(because of yarn.lock), so we can probably do it here too

Reply:
Yeah, I'm OK with that :+1:  `liferay-npm-scripts` is ours, we can do what we want with it.

I'll send another PR for that, then modify this one.